From f6194e5d2f57d58a4862f8572b5c78e8b470f18f Mon Sep 17 00:00:00 2001
From: crazy <crazy@frugalware.org>
Date: Tue, 15 Jan 2019 20:17:53 +0100
Subject: [PATCH 4/5] drop nobody from here

 well systemd assumes nobody/nogroup *is* gid/uid 65534
 however we do that from shadow package and they do have different gid/uid.
 for that matter we should not touch that but add some foobar user/group with
 whatever systemd expects
---
 sysusers.d/basic.conf.in | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/sysusers.d/basic.conf.in b/sysusers.d/basic.conf.in
index 20235f3699..1d26925513 100644
--- a/sysusers.d/basic.conf.in
+++ b/sysusers.d/basic.conf.in
@@ -8,9 +8,6 @@
 # The superuser
 u root    0     "Super User" /root
 
-# The nobody user for NFS file systems
-u @NOBODY_USER_NAME@  65534 "Nobody"     -
-
 # Administrator group: can *see* more than normal users
 g adm     -     -            -
 
-- 
2.20.1

